apptools/lru_cache/lru_cache.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
```

Comment:
We need the standard Enthought copyright boilerplate with an additional reference to the PSF License for the code that this was backported from.